corehq/util/itertools.py

```diff
@@ -0,0 +1,38 @@
+from __future__ import absolute_import
+from __future__ import unicode_literals
+
+from operator import lt, gt
+
+
+def merge(generator1, generator2, key=None, reverse=False):
```

Comment:
Looks like this is not used anymore, so it could be removed (although it's a cool implementation).

Reply:
:disappointed: 
https://github.com/dimagi/commcare-hq/pull/24240